src/v/cloud_storage/partition_manifest.cc

```diff
@@ -312,10 +312,17 @@ partition_manifest::get_start_kafka_offset() const {
             auto delta = iter->second.delta_offset;
             return _start_offset - delta;
         } else {
-            throw std::runtime_error(fmt_with_ctx(
-              fmt::format,
-              "can't translate start offset {} because the manifest is empty",
-              _start_offset));
+            // If start offset points outside a segment, then we cannot
+            // translate it.  If there are any segments ahead of it, then
+            // those may be considered the start of the remote log.
+            if (
+              !_segments.empty()
+              && _segments.begin()->second.base_offset >= _start_offset) {
```

Comment:
Is it possible for the first segment's base offset to be _below_ `_start_offset` if we failed to find it in our manifest and there are still segments? I could be wrong but I'd think `_start_offset` can only be moved to within range of `_segments` or exactly to its end. Fine with keeping this second condition as a safeguard, but I would like to understand if there are cases where we expect that condition to fire.